requirements/requirements.txt

```diff
@@ -1,7 +1,3 @@
-# fixes conflict with dateutil requirement
-# see: https://github.com/boto/botocore/commit/e87e7a745fd972815b235a9ee685232745aa94f9
-python-dateutil==2.8.0
-
 boto3~=1.0
```

Comment:
I'm wondering, do we even need `boto3` as a requirement?

Reply:
most likely not